client-guide/onboarding.html.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Onboarding
+---
+
+The onboarding process consists of six different steps:
+
+1. **Kick-off call:** During this call we will get to know each other. We will introduce ourselves,
+   our tools and our practices. We will also ask you to explain your project's goals and current
+   work methodology (if any) and what you expect from us.
+2. **Project evaluation:** After the kick-off call, we will ask for access to anything that might
+   help us figure out whether we're a good fit for each other. Depending on the stage of your
+   project, this might be a specification document or your existing codebase.
+3. **Negotiation and contract:** If we determine that we can provide value to your project, we will
+   send you an overview of the resources we expect will be needed to accomplish the required goals,
+   along with their cost. If everything looks good to you, we will then send a contract to sign.
+   [Here's a sample contract](/middleman/uploads/contract-draft.pdf).
+4. **Tool onboarding:** Once the contract is signed, we will ask you to provide access to all the
+   tools we will use to work on your project. Typically, these include tools such as Slack, GitHub
+   and anything else you use to organize your work.
+5. **Team meeting:** At this point, we will set up a meeting with your new team and ask you to bring
+   any members of your team that will interact frequently with the developers. The teams will get to
+   know each other and make sure there are no blockers to starting the work. 
+6. **Development kick-off:** Once we are all set, we will start working on your project!
+
+If you need to scale your team up, we will simply repeat steps 3-6.
```

Comment:
I think our contract already covers team scaling so maybe we don't need to re-negotiate and sign the contract again.